src/routers/users.py

```
@@ -1,14 +1,18 @@
+import logging
 from typing import Optional
```

Ruff (0.8.2)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)